### PR TITLE
Upgrade notes for Linkerd 2.9

### PR DESCRIPTION
**Pull request by alpeb**
To be merged only after 2.9 is released _and_ after #782 gets merged.

### CODE REVIEW

File: linkerd.io/content/2/tasks/upgrade.md [hunk 1]
```diff
@@ -171,6 +172,54 @@ Congratulation! You have successfully upgraded your Linkerd to the newer
...
+Users who installed the multicluster component in Linkerd 2.8.x and wish to
+upgrade to Linkerd 2.9 should follow the [upgrade multicluster
+instructions](/2/tasks/upgrade-multicluster/).
+
+### Ingress behavior changes
+
+In previous versions when you injected your ingress controller (Nginx, Traefik,
+Ambassador, etc), you would override the ingress' balancing/routing choices with
+Linkerd's (using Service Profiles, Traffic Splits, etc.). As of 2.9 the
+ingress's choices are honored instead (e.g. to preserve session-stickiness). If
```
Comment (adleong) [inline linkerd.io/content/2/tasks/upgrade.md:199]: might want to be a just a bit more explicit that this means that this means that per-route metrics will not be collected, traffic splits will not be honored, and retries/timeouts will not be applied.

File: linkerd.io/content/2/tasks/upgrade.md [hunk 2]
```diff
@@ -171,6 +172,54 @@ Congratulation! You have successfully upgraded your Linkerd to the newer
...
+you want to revert to the previous behavior, inject the proxy into the controller
```
Comment (adleong) [inline linkerd.io/content/2/tasks/upgrade.md:200]: Might want to say "into the ingress controller" just be extra clear

**Review by adleong — APPROVED** [review]
Nonblocking TIOLI suggestions

File: linkerd.io/content/2/tasks/upgrade.md [hunk 3]
```diff
@@ -171,6 +172,54 @@ Congratulation! You have successfully upgraded your Linkerd to the newer
...
+
+Linkerd 2.9 changes the way that some of the multicluster components work and
+are installed compared to Linkerd 2.8.x. Users installing the multicluster
+components for the first time with Linkerd 2.9 can ignore these instructions and
+instead refer directly to the [installing
+multicluster instructions](/2/tasks/installing-multicluster/).
+
+Users who installed the multicluster component in Linkerd 2.8.x and wish to
+upgrade to Linkerd 2.9 should follow the [upgrade multicluster
+instructions](/2/tasks/upgrade-multicluster/).
```
Comment (kleimkuhler) [inline linkerd.io/content/2/tasks/upgrade.md:192]: This doesn't seem to be a valid link and I'm not sure what it should be pointing at. Do we already have separate upgrade instructions for multicluster?

Comment (alpeb) [reply linkerd.io/content/2/tasks/upgrade.md:192]: That doc comes with #782. I've updated this PR's description instructing this should be merged only after #782 gets merged :+1:

Comment (kleimkuhler) [reply linkerd.io/content/2/tasks/upgrade.md:192]: Ah okay cool sounds good!

**Review by kleimkuhler — APPROVED** [review]

**Pull request merged**